### PR TITLE
Add DNS discovery 

### PR DESCRIPTION
**Pull request by atoulme**
Signed-off-by: Antoine Toulme <antoine@lunar-ocean.com>

## PR description
DNS discovery for goerli, rinkeby and mainnet to help speed up discovery of peers

## Changelog

- [ ] I thought about the changelog and included a [changelog update if required](https://wiki.hyperledger.org/display/BESU/Changelog).

### CODE REVIEW

**Comment by atoulme** [comment]
I tested this locally - the initial lag of peer discovery is erased.

File: ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/config/DiscoveryConfiguration.java [hunk 1]
```diff
@@ -27,6 +27,13 @@
 import java.util.stream.Stream;
 
 public class DiscoveryConfiguration {
+  public static String GOERLI_DISCOVERY_URL =
+      "enrtree://AKA3AM6LPBYEUDMVNU3BSVQJ5AD45Y7YPOHJLEF6W26QOE4VTUDPE@all.goerli.ethdisco.net";
```
Comment (davemec) [inline ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/config/DiscoveryConfiguration.java:31]: Should these three constants be declared `final`?

**Review by davemec — APPROVED** [review]
Looks good to me, just have one small comment regarding constants being `final`.

**Comment by atoulme** [comment]
Closing as work is underway elsewhere on ENRs.

File: ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/config/DiscoveryConfiguration.java (hunk 1 above)
Comment (atoulme) [reply ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/config/DiscoveryConfiguration.java:31]: added finals

**Pull request merged**